_emails/us/oregon/portland_defund_ppb.md

```diff
@@ -17,34 +17,44 @@ recipients:
 - sonia.schmanski@portlandoregon.gov
 - Toni.Anderson@portlandoregon.gov
 - Andrew.Bryans@portlandoregon.gov
-body: "I am a resident of Portland.\n\nIn May, Mayor Ted Wheeler released his proposed
-  budget for the City of Portland. This included a policing budget of $245,169,804,
-  which is over $5 million more than budgeted last year—even amongst city-wide cuts
-  to other departments due to Covid-19. The Portland Police Bureau (PPB), with Wheeler
-  at its helm as Commissioner, has been mired in controversy over the last year with
-  collusion with prominent white nationalist groups, shootings of Black people and
-  people perceived to be in mental health crisis, data uncovered on their racist gang
-  enforcement traffic stops, and the fact that a shameful 52% of people arrested are
-  houseless.\n\nI oppose any increases to the PPB budget. Mayor Wheeler's 2019-20
-  proposed budget was increased by an additional half million dollars before adoption,
-  and even then the PPB has overspent the increased budget by 1.4 million to date.
-  We anticipate that PPB will attempt a similar increase this year, pushing their
-  budget even higher than the proposed $245 million, thus we stand even more firmly
-  against this increase. \n\nAdditionally and critically, Wheeler's budget ELIMINATES
-  $1.8 million to fund the police body camera program. More money for policing but
-  less money for police accountability is completely unacceptable.\n\nThe police budget
-  increases regularly while other services are cut. I can see the ways in which policing
-  continues to perpetuate violence against Black, brown, houseless people, and against
-  people in mental health crisis. While Wheeler's proposal represents a slight decrease
-  from the PPB proposal, the PPB is a department whose reach into Portland communities
-  needs to be curtailed through reducing the budget, not by increasing it.\n\nMore
-  money for policing but less money for police accountability is completely unacceptable
-  at this moment in history. The city, the state, and country are all watching. And
-  so am I. I call on the Mayor, the Commissioners, and all other City Council Members
-  to use their powers to continue to reduce the budget and pledge to increase Police
-  accountability and transparency. I promise that you will lose my support and my
-  vote if you aren't willing to make a public and vocal stand to help defund the PPB
-  and increase their accountability and transparency.\n\nThank you for your time,\n[YOUR
-  NAME]\n[YOUR ADDRESS]\n[YOUR EMAIL]\n[YOUR PHONE NUMBER]"
+body: "Hello,
+
+My name is [YOUR NAME] and I am a resident of Portland. I would like to see us as a city committed to disbanding the Portland Police Bureau (PPB) immediately. We must get over this hurdle so that we can start to make real change.
```

Comment:
```suggestion
body:  |-
Hello,

My name is [YOUR NAME] and I am a resident of Portland. I would like to see us as a city committed to disbanding the Portland Police Bureau (PPB) immediately. We must get over this hurdle so that we can start to make real change.
```